README.md

```diff
@@ -70,16 +70,16 @@ The table below captures the state of various features and their levels of suppo
 | Feature                    | Node | Python | Java | .NET | Go   | Ruby |
 | -------------------------- | :--: | :----: | :--: | :--: | :--: | :--: |
 | OpenTelemetry collector    |  +   |  +     |  +   |  +   |  +   |      |
-| Custom context propagation |      |        |  -   |  -   | N/A  |      |
-| X-Ray Env Var Span Link    |      |        |  -   |  -   | N/A  |      |
-| Semantic Conventions^      |      |        |  +   |  +   | N/A  |      |
-| - Trace General^<sup>[1]</sup>           |      |        |  +   |  +   | N/A  |      |
-| - Trace Incoming^<sup>[2]</sup>          |      |        |  +   |  +   | N/A  |      |
-| - Trace Outgoing^<sup>[3]</sup>          |      |        |  +   |  +   | N/A  |      |
+| Custom context propagation |      |  -     |  -   |  -   | N/A  |      |
+| X-Ray Env Var Span Link    |      |  +     |  -   |  -   | N/A  |      |
```

Comment:
This isn't implemented yet in any language afaik since the spec was just merged

Reply:
Updated